examples_tests/06.MeshLoaders/mesh.vert

```diff
@@ -1,14 +1,18 @@
-#version 330 compatibility
-uniform vec3 cameraPos;
-uniform mat4 MVP;
+#version 450 core
+
+layout (std140, binding = 0) uniform UBO
```

Comment:
how does shaderc compile this without descriptor sets?

Reply:
if set is not specified than set=0 if that's what u mean

Reply:
@Crisspl TODO

Reply:
@Crisspl still a TODO.

Reply:
not relevant since this example doesnt really exist in any offcial form

Reply:
!? we should really be explicitly specifying sets

Reply:
its not a working example, dont even look at this code

Reply:
@AnastaZIuk  for future reference.